### PR TITLE
#1 'Footer is Broken' corrected.

### PR DESCRIPTION
**Pull request by SmruthyJoseph**
#1 '**Footer is Broken**' Corrected.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -45,7 +45,16 @@
...
+       .footer-size{
+           font-size: 25px;
+       }
+       .footer i{
+           font-size: 30px;
+       }
+       footer, .copyright p{
+           font-size: 17px;
+       }
+
```
Comment (AbhijithHaridas) [inline index.html:57]: @SmruthyJoseph styles should not be included in the template using a style tag. Please move this to the CSS file

**Review by AbhijithHaridas — CHANGES_REQUESTED** [review]
@SmruthyJoseph please correct the changes

File: index.html (hunk 1 above)
Comment (AbhijithHaridas) [reply index.html:57]: Why did this happen?

Comment (SmruthyJoseph) [reply index.html:57]: Ok, I will correct it.

Comment (SmruthyJoseph) [reply index.html:57]: The 'Google Font's size ' can't change if I put those codes in style.css. That's why I  put font styling in index page. Sorry I will correct them.

**Comment by SmruthyJoseph** [comment]
#1 'Footer is Broken' Corrected.

**Review by AbhijithHaridas — CHANGES_REQUESTED** [review]
@SmruthyJoseph Why there are so many file changes that are not part of this issue?

**Comment by SmruthyJoseph** [comment]
> @SmruthyJoseph Why there are so many file changes that are not part of this issue?
Sorry, I pushed all files accidentally.

**Comment by AbhijithHaridas** [comment]
This should not happen again and I need this corrected today itself

**Comment by SmruthyJoseph** [comment]
> This should not happen again and I need this corrected today itself

Ok.

**Comment by AbhijithHaridas** [comment]
### Disqualified

**Pull request closed without merging**